dags/pytorch_xla/pytorchxla-torchbench-release.py

```diff
@@ -32,106 +32,103 @@
       extraFlags=" ".join(torchbench_extra_flags),
   )
 
   # Running on V5E
   config.get_torchbench_tpu_config(
       tpu_version=resource.TpuVersion.V5E,
       tpu_cores=4,
-      project=resource.Project.TPU_PROD_ENV_AUTOMATED,
-      tpu_zone=resource.Zone.US_EAST1_C,
+      project=resource.Project.CLOUD_ML_BENCHMARKING,
+      tpu_zone=resource.Zone.US_WEST1_C,
       runtime_version=resource.RuntimeVersion.V2_ALPHA_TPUV5_LITE,
-      network=resource.V5_NETWORKS,
-      subnetwork=resource.V5E_SUBNETWORKS,
-      test_version=test_version,
+      network=resource.BM_NETWORKS,
+      subnetwork=resource.V5E_BM_SUBNETWORKS,
       time_out_in_min=1600,
       model_name=model,
+      reserved=False,
       extraFlags=" ".join(torchbench_extra_flags),
   )
 
   # Running on V100 GPU
-  config.get_torchbench_gpu_config(
-      machine_type=resource.MachineVersion.N1_STANDARD_8,
-      image_project=resource.ImageProject.DEEP_LEARNING_PLATFORM_RELEASE,
+  config.get_torchbench_gpu_gke_config(
+      machine_type=resource.MachineVersion.N1_STANDARD_16,
       image_family=resource.ImageFamily.COMMON_CU121_DEBIAN_11,
       accelerator_type=resource.GpuVersion.V100,
-      count=1,
-      gpu_zone=resource.Zone.US_CENTRAL1_C,
-      nvidia_driver_version="535.86.10",
-      test_version=test_version,
+      count=2,
+      gpu_zone=resource.Region.US_CENTRAL1,
+      project_name=resource.Project.CLOUD_ML_BENCHMARKING,
+      cluster_name="benchmarking-gpu-uc1",
       model_name=model,
       time_out_in_min=1600,
       extraFlags=" ".join(torchbench_extra_flags),
   ).run()
 
   # Running on A100 GPU
-  config.get_torchbench_gpu_config(
+  config.get_torchbench_gpu_gke_config(
       machine_type=resource.MachineVersion.A2_HIGHGPU_1G,
-      image_project=resource.ImageProject.DEEP_LEARNING_PLATFORM_RELEASE,
       image_family=resource.ImageFamily.COMMON_CU121_DEBIAN_11,
       accelerator_type=resource.GpuVersion.A100,
       count=1,
-      gpu_zone=resource.Zone.US_CENTRAL1_F,
-      nvidia_driver_version="535.86.10",
-      test_version=test_version,
+      gpu_zone=resource.Region.US_CENTRAL1,
+      project_name=resource.Project.CLOUD_ML_BENCHMARKING,
+      cluster_name="benchmarking-gpu-uc1",
       model_name=model,
       time_out_in_min=1600,
       extraFlags=" ".join(torchbench_extra_flags),
   ).run()
 
   # Running on H100 GPU
-  # Note: H100 must use ssd.
-  config.get_torchbench_gpu_config(
+  config.get_torchbench_gpu_gke_config(
       machine_type=resource.MachineVersion.A3_HIGHGPU_8G,
-      image_project=resource.ImageProject.DEEP_LEARNING_PLATFORM_RELEASE,
       image_family=resource.ImageFamily.COMMON_CU121_DEBIAN_11,
       accelerator_type=resource.GpuVersion.H100,
       count=8,
-      gpu_zone=resource.Zone.US_CENTRAL1_A,
-      nvidia_driver_version="535.86.10",
-      test_version=test_version,
+      gpu_zone=resource.Region.US_CENTRAL1,
+      project_name=resource.Project.CLOUD_ML_BENCHMARKING,
```

Comment:
i guess from r2.4 onward, we'll use the new project?

Reply:
Yes, we are switching the GPU test to GKE in the new project. There is plenty of resource left there.